doc/users/interactive.rst

```diff
@@ -9,40 +9,41 @@
 .. toctree::
 
 
-When working with data it is often invaluable to be able to interact
-with your plots. In many cases the built in pan/zoom and mouse-location
-tools are sufficient, but you can also use the Matplotlib event system
-to build customized data exploration tools.
+When working with data, interactivity can be invaluable. The pan/zoom and 
+mouse-location tools built into the Matplotlib GUI windows are often sufficient, but 
+you can also use the event system to build customized data exploration tools.
 
 Matplotlib ships with :ref:`backends <what-is-a-backend>` binding to
 several GUI toolkits (Qt, Tk, Wx, GTK, macOS, JavaScript) and third party
 packages provide bindings to `kivy
 <https://github.com/kivy-garden/garden.matplotlib>`__ and `Jupyter Lab
-<https://github.com/matplotlib/ipympl>`__.  For the figures to be
-"live" the GUI event loop will need to be integrated with your prompt. The
-simplest way is to use IPython (see :ref:`below <ipython-pylab>`).
+<https://github.com/matplotlib/ipympl>`__.  For the figures to be interactive, 
+the GUI event loop will need to be integrated with your interactive prompt. 
+We recommend using IPython (see :ref:`below <ipython-pylab>`).
 
-The `.pyplot` module provides functions for explicitly creating
-Figures that include interactive tools, a toolbar, a tool-tip, and
-:ref:`key bindings <key-event-handling>` ready to go:
+The `.pyplot` module provides functions for explicitly creating figures 
+that include interactive tools, a toolbar, a tool-tip, and
+:ref:`key bindings <key-event-handling>`:
 
-`.pyplot.figure`
+* `.pyplot.figure`
     Creates a new empty `.figure.Figure` or selects an existing figure
 
-`.pyplot.subplots`
+* `.pyplot.subplots`
     Creates a new `.figure.Figure` and fills it with a grid of `.axes.Axes`
 
 `.pyplot` has a notion of "The Current Figure" which can be accessed
 through `.pyplot.gcf` and a notion of "The Current Axes" accessed
 through `.pyplot.gca`.  Almost all of the functions in `.pyplot` pass
 through the current `.Figure` / `.axes.Axes` (or create one) as
-appropriate.  Matplotlib keeps a reference to all of the open figures
+appropriate.  
+
+Matplotlib keeps a reference to all of the open figures
 created this way so they will not be garbage collected.  You can close
```

Comment:
Looking at this again, "this way" previously had (when this was one paragraph) some tie in to the 'Current Figure'/'Current Axes' textually, but it doesn't actually in code. So I think this line needs a clarification of what 'this way' is, or this paragraph needs to go before the paragraph on 'current'-ness.